css/main.css

```diff
@@ -0,0 +1,964 @@
+  background-color: #fff;
+  margin-top: -2.1em;
+  -moz-clip-path: none;
+  -webkit-clip-path: polygon(50% 60%, 40% 0, 60% 0);
+  clip-path: polygon(50% 60%, 40% 0, 60% 0);
+}
+
+.b-white::before {
+  display: block;
+  height: 4em;
+  content: " ";
+  background-color: #fd6632;
+  margin-top: -2.1em;
+  -webkit-clip-path: polygon(50% 60%, 40% 0, 60% 0);
+  clip-path: polygon(50% 60%, 40% 0, 60% 0);
+}
+
+.b-black::before {
+  display: block;
+  height: 4em;
+  content: " ";
+  background-color: #fff;
+  margin-top: -2.1em;
+  -webkit-clip-path: polygon(50% 60%, 40% 0, 60% 0);
+  clip-path: polygon(50% 60%, 40% 0, 60% 0);
+}
+
+.b-black-white::before {
+  display: block;
+  height: 4em;
+  content: " ";
+  background-color: #252629;
+  margin-top: -2.1em;
+  -webkit-clip-path: polygon(50% 60%, 40% 0, 60% 0);
+  clip-path: polygon(50% 60%, 40% 0, 60% 0);
+}
+
+.l-flex-container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-content: center;
+  align-items: center;
+}
+
+.l-flex-item {
+  text-align: center;
+  margin: auto;
+  max-width: 85%;
+  padding-top: 2.5%;
+  padding-bottom: 2.5%;
+}
+
+.l-flex-contact {
+  padding-top: 0;
+  padding-bottom: 0;
+}
+
+.l-flex-child {
+  margin-bottom: 10%;
+}
+
+.l-flex-child-p {
+  margin-top: 10%;
+}
+
+.l-flex-prices {
+  width: 80%;
+  margin: auto;
+}
+
+.l-slider {
+  max-width: 100%;
+}
+
+.l-client {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  justify-content: space-between;
+  align-content: space-between;
+  align-items: center;
+}
+
+.l-client-item {
+  max-width: 85%;
+  order: 2;
+  text-align: center;
+  margin: auto;
+}
+
+.l-footer {
+  margin-bottom: 5%;
+  margin-top: 2%;
+}
+
+/*=== STYLES ===*/
+body {
+  font-family: 'Helvetica', 'Arial', Sans-serif;
+  font-weight: 400;
+}
+
+nav {
+  font-family: 'Open Sans', 'Helvetica', Sans-serif;
+  font-weight: 600;
+}
+
+ul {
+  list-style: none;
+  padding: 0;
+}
+
+td {
+  background-color: #ffffff;
+  color: #2e2e2e;
+  font-size: 1.5em;
+}
+
+article {
+  padding: 2em 1em;
+}
+
+th, td {
+  padding: .25em 1em;
+  margin: .8em auto;
+}
+
+em {
+  font-style: none;
+}
+
+h2 {
+  width: 100%;
+  display: inline-block;
+}
+
+h2:before, h2:after {
+  content: " ";
+  display: inline-block;
+  border-top: solid 1px white;
+  width: 18%;
+  height: 1px;
+  z-index: 1;
+  position: relative;
+  margin: .25em;
+  white-space: nowrap;
+}
+
+.b-white h2:before, .b-white h2:after {
+  border-top: solid 1px black;
+}
+
+.h3-lines {
+  width: 100%;
+  display: inline-block;
+}
+
+.h3-lines:before, .h3-lines:after {
+  content: " ";
+  display: inline-block;
+  border-top: solid 1px white;
+  width: 18%;
+  height: 1px;
+  z-index: 1;
+  position: relative;
+  margin: .325em;
+  white-space: nowrap;
+}
+
+.b-white .h3-lines:before, .b-white .h3-lines:after {
+  border-top: solid 1px black;
+}
+
+.s-contact-info a, .s-contact-info a:hover, .s-contact-info a:visited {
```

Comment:
When your selector strings start getting to this length, it's best to start breaking them out to one on a line:

```css
.s-contact-info a,
.s-contact-info a:hover,
.s-contact-info a:visited {
  /* ... */
}
```